packages/flutter/lib/src/services/asset_bundle.dart

```diff
@@ -63,7 +64,7 @@ class MojoAssetBundle extends AssetBundle {
   }
 
   AssetBundleProxy _bundle;
-  Map<String, ImageResource> _imageCache = new Map<String, ImageResource>();
+  Map<String, ui.Image> _imageCache = new Map<String, ui.Image>();
```

Comment:
A ui.Image is a bag of bits. The data coming from the mojo pipe might have more than one bag of bits (e.g. multiple frames of an animated GIF). We don't yet handle that though.

cc @abarth for his plans for how to handle animated GIFs.